src/ucp/core/ucp_context.c

```diff
@@ -745,6 +742,30 @@ static void ucp_resource_config_str(const ucp_config_t *config, char *buf,
     }
 }
 
+static void ucp_fill_sockaddr_aux_tls_config(ucp_context_h context,
+                                             const ucp_config_t *config)
+{
+    unsigned num_sockaddr_aux_tls, idx;
+    ucp_rsc_index_t tl_id;
+
+    context->config.sockaddr_aux_rscs.bitmap   = 0;
+    context->config.sockaddr_aux_rscs.num_rscs = config->sockaddr_aux_tls.count;
```

Comment:
sockaddr_aux_rscs.num_rscs is not required. you only need the bitmap
no need for the wrapper struct "sockaddr_aux_rscs"